lighthouse-core/audits/non-composited-animations.js

```diff
@@ -27,6 +27,16 @@ const UIStrings = {
     =1 {Unsupported CSS Property: {properties}}
     other {Unsupported CSS Properties: {properties}}
   }`,
+  /** Name of a compositor failure reason where the transform being animated depends on the box size. */
+  transformDependsBoxSize: 'Transform related property depends on box size',
```

Comment:
```suggestion
  transformDependsBoxSize: 'Transform-related property depends on box size',
```

I read this very differently on my first attempt and got confused 😆 maybe this would've helped me?  

[SE](https://english.stackexchange.com/questions/2908/should-i-use-related-or-related) seems to agree at least 🤷